testutils/call.go

```diff
@@ -64,6 +64,14 @@ func (f *FakeIncomingCall) RemotePeer() tchannel.PeerInfo {
 	return f.RemotePeerF
 }
 
+// CallOptions returns the incoming call options suitable for proxying a request.
+func (f *FakeIncomingCall) CallOptions() *tchannel.CallOptions {
+	return &tchannel.CallOptions{
+		ShardKey:        f.ShardKey(),
```

Comment:
I can set a dummy value for format (it's not a field on fake at the moment).

I can't set callerName since it's unexported and in a different package.